src/sql/parts/dashboard/common/dashboardPage.component.ts

```diff
@@ -208,7 +208,7 @@ export abstract class DashboardPage extends AngularDisposable {
 						configs = cb.apply(this, [configs, this.dashboardService, this.context]);
 					});
 					this._gridModifiers.forEach(cb => {
-						configs = cb.apply(this, [configs, this._originalConfig]);
```

Comment:
does this brake editing?

Reply:
The home tab doesn't call function. This function is for dashboard extension

---
In reply to: [178911833](https://github.com/Microsoft/sqlopsstudio/pull/1016#discussion_r178911833) [](ancestors = 178911833)